src/Data/Functor/Flip.purs

```diff
@@ -36,3 +36,11 @@ instance biapplicativeFlip :: Biapplicative p => Biapplicative (Flip p) where
 
 instance contravariantFlip :: Profunctor p => Contravariant (Flip p b) where
   cmap f (Flip a) = Flip (lcmap f a)
+
+instance semigroupoidFlip :: Semigroupoid p => Semigroupoid (Flip p)
+  where
```

Comment:
The code looks fine to me (I'm not an expert, so take that with a grain of salt), but I think you'd want to follow the rest of the file's formatting (I'm referring to `where` being on its own line here). Same with the `Category` instance.

Reply:
Good catch, thanks!